README.md

```diff
@@ -25,6 +25,13 @@ If you like to share your app with rest of the Rockstor community, open a pull r
 1. One Rock-On per pull request please. If you are working on multiple apps, separate them out. It will make testing and merging a lot more manageable.
 
 2. Add a comment to your pull request detailing how you've tested it out. More details the better as it will help ensure quality and benefit the whole community!
+3. We are trying to offer Rock-ons that are based on a multi-architecture docker image, i.e., it is available for the amd64 (Intel/AMD CPUs), armv7 and arm64 (ARM-based devices). While that might not always be possible, depending on the app you're interested, please keep it in mind and see whether you can select a docker image that has a multi-architecture manifest. When submitting, please add the supported architectures to the description (take a look at some of the more recent Rock-on submissions).
+
+## Adding vs. updating a Rock-on you want to submit to the repository
+While there are no hard and fast rules, you can follow these guidelines to decide whether it is better to update an existing rockon or submit a new one that contains substantial changes:
+1. If no Rock-on exists for your app already, create a new one. The obvious choice.
+2. Create a new Rock-on if one already exists for this project, but the new one will use a different docker image. E.g., an existing Rock-on uses an image from linuxserver.io, while the one you are interested in submitting uses an image created by the owner of the project. You can subsequently submit a proposal to deprecate the existing Rock-on. Reasons for that could be that the underlying docker container has not been maintained in a long time, or the new Rockon will have the same or more functionality and is more popular with the community.
+3. Update the existing Rock-on if the changes do not include the use of a different image. E.g., the Handbrake Rock-on was expanded with a few useful user parameters over time, but continued to use the same underlying docker image. Here it made more sense to update the existing Rock-on instead of submitting a new version.
```

Comment:
```suggestion

3. Update the existing Rock-on if the changes do not include the use of a different image. E.g., the Handbrake Rock-on was expanded with a few useful user parameters over time, but continued to use the same underlying docker image. Here it made more sense to update the existing Rock-on instead of submitting a new version.
```